python/vowpalwabbit/pyvw.py

```diff
@@ -1360,6 +1423,9 @@ def __init__(
         self.labelType: Optional[LabelType] = None
         if isinstance(labelType, LabelType):
             label_int = labelType.value
+            # Keep self.labelType as None if provided label is DEFAULT.
+            if labelType != LabelType.DEFAULT:
+                self.labelType = labelType
```

Comment:
@bassmang  @jackgerrits  For some reason we don't assign `self.labelType` in this block? Caught this while testing.

Reply:
Good catch!